meta-balena-common/recipes-core/systemd/systemd-serialgetty.bbappend

```diff
@@ -1,5 +1,15 @@
-do_install_prepend() {
-    sed -i '/^\[Unit\]/a ConditionPathExists=/var/volatile/development-features' ${WORKDIR}/serial-getty@.service
-    sed -i '/^\[Unit\]/a After=development-features.service' ${WORKDIR}/serial-getty@.service
-    sed -i '/^\[Unit\]/a Requires=development-features.service' ${WORKDIR}/serial-getty@.service
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append = " file://development-features.conf"
+
+do_install_append() {
+	if [ ! -z "${SERIAL_CONSOLES}" ] ; then
+		tmp="${SERIAL_CONSOLES}"
+		for entry in $tmp ; do
+			baudrate=`echo $entry | sed 's/\;.*//'`
```

Comment:
shellcheck would suggest replacing back ticks with `$()`

Reply:
This was only an initial draft, according to systemd manual it will eventually look at the template.d subdirectory so I will test that first.

Reply:
The loop was not required in the end so I removed it.